extensions/amp-next-page/1.0/page.js

```diff
@@ -224,6 +219,7 @@ export class Page {
     if (
       this.state_ === PageState.INSERTED ||
       this.state_ === PageState.FETCHING ||
+      this.state_ === PageState.LOADED ||
```

Comment:
Reading, it seems like they were allowed to refetch after it was loaded, and we're disallowing that now. Was it intentionally done before?

Reply:
It was unintentionally forgotten. We don't want a refetch during loading because it can cause duplicate insertion. After the `LOADED` state, it would either be `FAILED` or `INSERTED` and that's when we'd want to refetch or not.